aten/src/ATen/native/mps/operations/ReduceOps.mm

```diff
@@ -715,8 +715,8 @@ Tensor std_var_common_impl_mps(
     }
   }
 
-  bool use_correction = correction.has_value();
-  const auto correction_value = use_correction ? correction.value() : false;
+  bool use_correction = !(correction.has_value() && correction.value() == 0);
+  const auto correction_value = correction.value_or(1);
```

Comment:
By default PyTorch is using unbiased (correction=1) for std / var, unless explicitly passing correction=0 or unbiased=False.
https://github.com/pytorch/pytorch/pull/56398/files
"In PyTorch 1.9 and earlier std accepted an unbiased argument. If unbiased was True (the default) it implied correction=1 (the current default). If unbiased was False then it implied correction=0. The new correction parameter generalizes the deprecated unbiased parameter."